server/src/main/java/org/elasticsearch/common/util/BigArrays.java

```diff
@@ -380,9 +376,15 @@ public BigArrays(PageCacheRecycler recycler, @Nullable final CircuitBreakerServi
 
     // public for tests
     public BigArrays(PageCacheRecycler recycler, @Nullable final CircuitBreakerService breakerService, boolean checkBreaker) {
```

Comment:
can we remove this constructor and make call sites pass `CircuitBreaker.REQUEST` as a breaker name?